x-pack/plugins/lens/public/xy_visualization_plugin/plugin.tsx

```diff
@@ -4,12 +4,57 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { Registry } from '@kbn/interpreter/target/common';
+import { CoreSetup } from 'src/core/public';
 import { xyVisualization } from './xy_visualization';
 
+import {
+  renderersRegistry,
+  functionsRegistry,
+  // @ts-ignore
```

Comment:
What's this ignore for? Maybe this should be an explicit ignore, rather than a general one, so that it's clear what exact error we're ignoring. It probably should be `// @ts-ignore untyped dependency`

Reply:
This is about untyped dependencies, right. I added an explanatory comment